Source/ACE/Network/GameMessages/Messages/GameMessageUpdateMotion.cs

```diff
@@ -41,11 +42,50 @@ public GameMessageUpdateMotion(WorldObject animationTarget, MotionState newState
             else
                 Writer.Write(animationTarget.Sequences.GetCurrentSequence(Sequence.SequenceType.ObjectServerControl)); // Server_Control_Timestamp
 
-            ushort autonomous = newState.IsAutonomous ? (ushort)1 : (ushort)0;
+            ushort autonomous;
+            if (newState.IsAutonomous)
+                autonomous = 1;
+            else
+                autonomous = 0;
             Writer.Write(autonomous); // autonomous flag - 1 or 0.   I think this is set if you have are holding the run key or some other autonomous movement
             var movementData = newState.GetPayload(animationTarget);
             Writer.Write(movementData);
             Writer.Align();
         }
+
+        // TODO: Og II - Finish an overload for F74C 0x0006 - MoveToObject
+        /// <summary>
+        /// This method is used for server controled motion - such as move to object, turn to object etc.
+        /// </summary>
+        /// <param name="player"></param>
+        /// <param name="moveToTarget"></param>
+        /// <param name="newState"></param>
+        /// <param name="movementType"></param>
+        /// <param name="runRate"></param>
+        public GameMessageUpdateMotion(WorldObject player, WorldObject moveToTarget, ServerControlMotion newState, MovementTypes movementType, float runRate = 1.0f) : base(GameMessageOpcode.Motion, GameMessageGroup.Group0A)
+        {           
+            Writer.WriteGuid(player.Guid); // Object_Id (uint)
+            Writer.Write(player.Sequences.GetCurrentSequence(SequenceType.ObjectInstance)); // Instance_Timestamp
+            Writer.Write(player.Sequences.GetNextSequence(SequenceType.ObjectMovement)); // Movement_Timestamp
+
+            // Looking at the pcaps in training_hall_main_section.pcap - the key to server control timestamp seems to be the movement type
+            // TODO: Og II Reseach if it is all of the server control movement types - it may be all them them except MovementTypes.General
+
+             if (movementType == MovementTypes.MoveToObject | movementType == MovementTypes.TurnToObject)
```

Comment:
ServerControlTimestamp changes when Autonomous is 0, based on my research.  Basically when Autonomous is 0 its saying the server has directed the movement, meaning it is server controlled, I believe.